### PR TITLE
where now presence API

### PR DESCRIPTION
**Pull request by Xavrax**

### CODE REVIEW

File: src/dx/presence/mod.rs [hunk 1]
```diff
@@ -213,6 +213,46 @@ impl<T, D> PubNubClientInstance<T, D> {
...
+
+    ///
+    /// println!("User channels: {:?}", response);
+    ///
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn where_now(&self) -> WhereNowRequestBuilder<T, D> {
+        WhereNowRequestBuilder {
+            pubnub_client: Some(self.clone()),
```
Comment (parfeon) [inline src/dx/presence/mod.rs:252]: @Xavrax what do you think about default `user_id` usage here from `client` configuration?

**Review by parfeon — APPROVED** [review]
LGTM!

File: src/dx/presence/mod.rs (hunk 1 above)
Comment (Xavrax) [reply src/dx/presence/mod.rs:252]: Tbh. I prefer when API is more explicit but I'm not against the default value.

Comment (parfeon) [reply src/dx/presence/mod.rs:252]: We had this behaviour with default `user_id` in other SDKs for some endpoints.

Comment (parfeon) [reply src/dx/presence/mod.rs:252]: But it is totally fine as it now ;)

Comment (Xavrax) [reply src/dx/presence/mod.rs:252]: Okay. I don't want to make it different than in other SDKs. I will change that - but to keep git history not so complicated, I will change that in the nexts PRs. (I'm planning to make `state` and `no_std` PRs before release so I can make that change during one of those)

Comment (Xavrax) [reply src/dx/presence/mod.rs:252]: fixed.

**Pull request merged**